src/account/operations/syncing/options.rs

```diff
@@ -36,6 +43,10 @@ fn default_sync_aliases_and_nfts() -> bool {
     true
 }
 
+fn default_sync_only_most_basic_outputs() -> bool {
+    false
+}
+
 fn default_address_start_index() -> u32 {
     0
```

Comment:
maybe create some constant and return it here instead?  also for the other functions above^